cache/disk/lru.go

```diff
@@ -4,6 +4,26 @@ import (
 	"container/list"
 	"errors"
 	"fmt"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	gaugeCacheSizeBytes = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "bazel_remote_disk_cache_size_bytes",
+		Help: "The current number of bytes in the disk backend",
+	})
+
+	counterEvictedBytes = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "bazel_remote_disk_cache_evicted_bytes_total",
+		Help: "The total number of bytes evicted from disk backend, due to full cache",
+	})
```

Comment:
We expect this graph to stay at 0 until the cache is full, then increase with every Put that isn't short-circuited. What else should we be looking for in this graph?

Reply:
Yes. After reaching the limit, it provides similar information as gaugeCacheSizeBytes does before reaching the limit.

This value also seems to be intuitive to understand for my users. (they don't know about short-circuited PUT, etc)

Reply:
I suppose it might be interesting to observe changes that invalidate large parts of the cache.